src/main/java/org/usfirst/frc/team2473/robot/subsystems/Elevator.java

```diff
@@ -0,0 +1,90 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc.team2473.robot.subsystems;
+
+import org.usfirst.frc.team2473.framework.Devices;
+import org.usfirst.frc.team2473.robot.RobotMap;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * This class contains all components of the robot necessary for driving.
+ */
+public class Elevator extends Subsystem {
+    
+    public enum ElevatorPosition {
+        BASE(100), FIRST(500), SECOND(1000), THIRD(1500);
```

Comment:
What units are these values? How were they determined?

Reply:
Units are encoder ticks. These are just dummy values, they'll be determined when we actually get to run the elevator.

Reply:
Added comment, will push